pybwa/libbwaaln.pyx

```diff
@@ -4,6 +4,7 @@ from pathlib import Path
 from typing import List
 from fgpyo.sequence import reverse_complement
 
+import pysam
```

Comment:
nit: by convention in this repo we import just the functions and constants we need.  I should probably document that :P

Can you add to the `from pysam` on line 11 below?

Then update:

```python
_BWA_ALN_TO_PYSAM_CIGAR_OPERATOR[FROM_M] = CMATCH
_BWA_ALN_TO_PYSAM_CIGAR_OPERATOR[FROM_I] = CINS
_BWA_ALN_TO_PYSAM_CIGAR_OPERATOR[FROM_D] =CDEL
_BWA_ALN_TO_PYSAM_CIGAR_OPERATOR[FROM_S] =CSOFT_CLIP
```

```python
from pysam import FastxRecord, AlignedSegment, qualitystring_to_array, CMATCH, CINS, CDEL, CSOFT_CLIP
```